tests/lib/prepare-restore.sh

```diff
@@ -359,11 +359,16 @@ prepare_project() {
     go get ./tests/lib/fakedevicesvc
     go get ./tests/lib/systemd-escape
 
-    # disable journald rate limiting
-    mkdir -p /etc/systemd/journald.conf.d/
+    # Disable journald rate limiting
+    # Check the RateLimitInterval key which id different depending on the systemd version
```

Comment:
`s/id/is/`

The commit that added `RateLimitIntervalSec` is https://github.com/systemd/systemd/commit/f0367da7d1a61ad698a55d17b5c28ddce0dc265a It looks like they retained backwards compatibility so we still may be good just setting `RateLimitInterval=0` for all systems and leave a huge note in the code pointing to this commit.